tests/sentry/utils/performance_issues/test_http_overhead_detector.py

```diff
@@ -260,3 +260,23 @@ def test_detect_other_location(self):
                 evidence_display=[],
             )
         ]
+
+    def test_none_request_start(self):
+        url = "https://example.com/api/endpoint/123"
+        event = _valid_http_overhead_event("/api/endpoint/123")
+
+        # Include an invalid span to ensure it's not processed
+        span = create_span(
+            "http.client",
+            desc=url,
+            duration=1000,
+            data={
+                "url": url,
+                "network.protocol.version": "1.1",
+                "request_start": None,
```

Comment:
```suggestion
                "http.request.request_start": None,
```
Sorry, I had this wrong in my suggested code in the original review. Tested this locally to confirm it repros the original bug.